service/controller/clusternamespace/resource/clusternamespaceresources/create_test.go

```diff
@@ -0,0 +1,158 @@
+package clusternamespaceresources
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/giantswarm/k8sclient/v7/pkg/k8sclienttest"
+	"github.com/giantswarm/micrologger/microloggertest"
+	security "github.com/giantswarm/organization-operator/api/v1alpha1"
+	"github.com/google/go-cmp/cmp"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgofake "k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/kubernetes/scheme"
+	clientfake "sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func Test_EnsureCreated(t *testing.T) {
+	tests := []struct {
+		name                 string
+		namespaces           []*corev1.Namespace
+		organization         *security.Organization
+		roleBindings         []*rbacv1.RoleBinding
+		expectedRoleBindings []*rbacv1.RoleBinding
+	}{
+		{
+			name: "flawless",
```

Comment:
Would be good to have another namespace to ensure we are not adding rolebindings to non-org namespaces.

Reply:
good point, will add it as well

Reply:
@MarcelMue it's done, see the `expectedRoleBindingsNum` map